test/e2e/fixtures.go

```diff
@@ -428,7 +428,7 @@ func exportLogs(tb testing.TB, data *TestData, logsSubDir string, writeNodeLogs
 		writeVMAgentLog(cmd, testOptions.linuxVMs)
 	}
 	if testOptions.windowsVMs != "" {
-		cmd := "cat c:/antrea-agent/antrea-agent.log"
+		cmd := "cat c:/antrea-agent/logs/antrea-agent.log"
```

Comment:
where can we find a reference for the log path? I searched "c:/antrea-agent/logs" but got nothing.

Reply:
https://github.com/antrea-io/antrea/blob/1b87db8025a42c4873b3beb469b357b5677db8f1/hack/externalnode/install-vm.ps1#L46 You can find the reference in this line.

Reply:
I think it is because this https://github.com/antrea-io/antrea/blob/main/hack/externalnode/install-vm.ps1#LL46C1-L46C61 because the test-vm.sh has called install-vm.ps1 to install antrea-agent on Windows VM ?

Reply:
got it, thanks